utxo/src/cache.rs

```diff
@@ -88,11 +106,11 @@ pub struct UtxosCache<'a> {
     memory_usage: usize,
 }
 
-impl<'a> UtxosCache<'a> {
+impl<'a, P: UtxosView> UtxosCache<'a, P> {
     #[cfg(test)]
-    pub fn new_for_test(best_block: Id<GenBlock>, view: &'a dyn UtxosView) -> Self {
+    pub fn new_for_test(best_block: Id<GenBlock>, parent: UtxosViewCow<'a, P>) -> Self {
```

Comment:
not directly related, but I think we can remove this function at all. The only difference that it creates cache with specific best block id. Default id::zero from `EmptyUtxosView` is no different for our tests. If needed one can always call `set_best_block`. But without this function `UtxosViewCow` wouldn't be exposed to the rest of the code and will make things cleaner.

Reply:
Done